### DISCUSSION TITLE
PXE menu for specific system

### DISCUSSION THREAD

**Discussion by y2kenny** [cobbler/cobbler#2585] [Q&A]
### Version

```
Cobbler 2.8.5
  source: ?, ?
  build time: Tue Oct 15 01:56:01 2019
```
### Question

For any system not specified in Cobbler, if it is PXE booted, it will see a list of profile available to use.  But if I add a system to Cobbler, I have to specify a profile since selecting a 'None' profile will cause a sync error per #1510.  Is there a way for me to specify a system in Cobbler but still have it see the default pxelinux.cfg (so that user can manually pick the profile to PXE boot?)

**Comment by SchoolGuy** [comment]
No. What you want is per definition not possible. @jmaas also answered this back then and in this regard nothing has changed. If you want to change the profile you need to change it in the system also.

In my eyes this is pretty reasonable: Either you want your system managed and want to know the state of it or you have an unmanaged system. Everything in between will only cause trouble in my eyes.

**Comment by y2kenny** [comment]
@SchoolGuy So the way I was thinking of using it is this: Cobbler manages things like IP addresses, domain-name, etc. but the users of the system get to pick the OS/distros they want to install from the PXE menu.  I don't think this is too crazy of a use case.  If there are other ways to achieve the same functionality I am certainly open to it.

**Comment by SchoolGuy** [comment]
Fine. I see your point now. But for now I can't prioritize this since there is so much on the table. But I won't forget it.

**Comment by y2kenny** [comment]
@SchoolGuy Understood and I appreciate all your work.  I will look into contributing a PR when I have time.
